english/lego/lego.md

```diff
@@ -0,0 +1,19 @@
+The mallow man seems nice so you chat to him. He decides he likes you too, so he gets out his Lego's. 
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.